spec/presenters/tslr_claim_csv_row_spec.rb

```diff
@@ -0,0 +1,54 @@
+require "rails_helper"
+require "csv"
```

Comment:
This require needs to go in the file for the class itself, otherwise the `CSV` library isn't loaded outside of the test environment.